FirebaseUI.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,14 @@
 	objects = {
```

Comment:
The `FirebaseTwitterAuthUI` target needs to have a target dependency on `FirebaseAuthUI` or it won't compile.
<img width="730" alt="screen shot 2016-09-15 at 12 37 31 pm" src="https://cloud.githubusercontent.com/assets/4529021/18564639/4b029e60-7b41-11e6-9168-1e6b3387669d.png">